students/14th/eunjungko/style/login.css

```
@@ -0,0 +1,48 @@
+body {
+  display: flex;
+  justify-content: center;
+  background-color: #fafafa;
+}
+
+.loginWrapper {
+  width: 390px;
+  height: 420px;
+  margin: 30px auto;
+  padding: 30px 45px;
+  border: 1px solid #e8e8e8;
+  background-color: #fff;
+}
+.logo {
```

```suggestion
}

.logo {
```
구분될 수 있는 요소간 한줄 씩 띄워주세요

---

네! 수정했습니다 :)